blog.html

```diff
@@ -0,0 +1,251 @@
+                        <li>List name</li>
+                    </ol>
+                    <div class="blog__blockquote">
+                        <blockquote cite="https://">
+                            <p class="blog__blockquote--text">The first huge metaverse arena to arrange live sports, hang out and even make bets and wagers</p>
+                        </blockquote>
+                        <p class="blog__blockquote--rating">8.5/ 10<cite>Markout Corporation</cite></p>
+                    </div>
+            </article>
+
+            <figure class="blog__photo">
+                <img src="images/blog-photo.jpg" alt="Basilisk NFT">
+                <figcaption class="blog__photo-caption">Photo caption</figcaption>
+            </figure>
+
+            <figure class="blog__photo">
+                <img src="images/blog-photo.jpg" alt="Basilisk NFT">
+                <figcaption class="blog__photo-caption">Photo caption</figcaption>
+            </figure>
+
+            <article class="blog__info">
+                <h3 class="intro__subtitle">NFTs are digital assets that represent real-world objects</h3>
+                <p class="blog__text">Setting up your avatar and personalizing your nickname on Binance is quick and easy. Users can go to binance.com under 
+                    Dashboard > Settings > My Profile > Avatar and Nickname. For those of you who own NFT assets, you can select the NFT you
+                    want to use from your collection as your avatar to personalize your profile. For users who do not own any NFT avatars, 
+                    check out the NFT avatar collections page to get started.</p>
+
+                    <p>Alternatively, if you’re on nft.binance.com, go to User Center > Settings > Basic > Set Avatar and Nickname, and you’ll be 
+                    redirected to the main site. Binance NFT Creators have the option of setting their Avatars on nft.binance.com. Once you set 
+                    your new unique nickname, it will be standardized and used across the Binance ecosystem.</p>
+
+                <h3 class="intro__subtitle">NFT Avatar Generators</h3>
+                <p>You can find a range of NFT marketplaces where you can buy existing NFTs or list your own NFTs for sale.</p>
+
+                <div class="blog__tags">
+                    <span class="blog-article__tags-title">Tags:</span>
+                    <a href="#" class="blog-article__tag">NFT</a>
+                    <a href="#" class="blog-article__tag">ICO</a>
+                    <a href="#" class="blog-article__tag">Crypto</a>
+                </div>
+
+                <div class="blog__share">
+                    <span class="blog-article__share-title">Share:</span>
+                    <a href="#" class="blog-article__share-link">Twitter</a>
+                    <a href="#" class="blog-article__share-link">Facebook</a>
+                    <a href="#" class="blog-article__share-link">LinkedIn</a>
```

Comment:
It's ul > li > a > img 
Class for ul could be .social-networks.social-networks--alt   or share-social-networks